packages/audit/tests/test_audit_hooks.py

```
@@ -0,0 +1,191 @@
+
+class TestDetectRoute:
+    def test_bash_codex(self) -> None:
+        data = {"tool_name": "Bash", "tool_input": {"command": "codex exec --model gpt-5 'hello'"}}
+        route, excerpt, tool = audit_route.detect_route(data)
+        assert route == "bash:codex"
+        assert "codex" in excerpt
+        assert tool == "Bash"
+
+    def test_bash_gemini(self) -> None:
+        data = {"tool_name": "Bash", "tool_input": {"command": "gemini -m model -p 'query'"}}
+        route, excerpt, tool = audit_route.detect_route(data)
```

Ruff (0.15.9)

[warning] Unpacked variable `excerpt` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)